svgdigitizer/models/exceptions.py

```diff
@@ -0,0 +1,101 @@
+r"""
```

Comment:
I had a file like this in mind, yes. What do you mean when you say `models`? Maybe just make it `svgdigitizer/exceptions.py`?

Reply:
TBH I stored the file in a folder according to some suggested structures I found online. Intuitively, I would have also put it into the parent directory, while this is probably more reasonable for larger projects.